src/lib/comms/xbee/xbee_driver.cpp

```diff
@@ -125,6 +126,50 @@ void jaiabot::comms::XBeeDriver::startup(const goby::acomms::protobuf::DriverCon
                          config_extension().hub_id(), config_extension().fleet_id());
     }
 
+    // if simulation, add XBee to Simulated XBee Network
+    if (is_in_sim)
```

Comment:
Looks fine to me.

I'd write it using something more C++ oriented (likely Boost Asio) but for this simple purpose raw sockets are easy enough to write and understand.